frontend/src/metabase/components/LabelIcon/LabelIcon.module.css

```diff
@@ -1,16 +1,16 @@
-:local(.colorIcon) {
+.colorIcon {
   composes: inline-block from "style";
   width: 18px;
   height: 18px;
   border-radius: 3px;
 }
 
-:local(.emojiIcon) {
+.emojiIcon {
   composes: text-centered from "style";
   font-size: 20px;
 }
 
-:local(.icon) {
-  composes: transition-color from "style";
+.icon {
+  transition: color 0.3s linear;
```

Comment:
you're inlining this property everywhere. Should you still keep that as a CSS module rule so you could compose it like other rules?

Reply:
that's why I suggested separating removal of composes